util.py

```diff
@@ -300,7 +300,7 @@ def instanciate_delayed(o):
 
 def get_executable_path(
     path: Optional[tk.Path],
-    gs_member_name: str,
+    gs_member_name: Optional[str],
```

Comment:
this variable seems unused

Reply:
It is not right? c.f. Line 337

Reply:
Oh, I saw something different, forget about this